docs/intro.rst

```diff
@@ -4,30 +4,31 @@
 Introduction
 ============
 
-*dwave-system* enables easy incorporation of the D-Wave system as a
-:term:`sampler`---the component used to find variable values that minimize the binary
-quadratic model (BQM) representing a problem---in the typical Ocean
-:std:doc:`problem-solving procedure <oceandocs:overview/solving_problems>`\ :
-
-1. Formulate the problem as a BQM.
-2. Solve the BQM with a sampler.
-
-You can incorporate the D-Wave system in either a hybrid quantum-classical solution,
-using :class:`~dwave.system.samplers.LeapHybridSampler()` or
-:std:doc:`dwave-hybrid <oceandocs:docs_hybrid/sdk_index>` samplers such as :code:`KerberosSampler()`,
-or directly using :class:`~dwave.system.samplers.DWaveSampler()`.
+*dwave-system* enables easy incorporation of the D-Wave system as a :term:`sampler`
+in either a hybrid quantum-classical solution, using
+:class:`~dwave.system.samplers.LeapHybridSampler()` or
+:std:doc:`dwave-hybrid <oceandocs:docs_hybrid/sdk_index>` samplers such as
+:code:`KerberosSampler()`, or directly using :class:`~dwave.system.samplers.DWaveSampler()`.
+
+.. note:: For applications that require detailed control on communication with the remote
+    compute resource (a D-Wave QPU or Leap's hybrid solvers), see
+    :std:doc:`dwave-cloud-client <oceandocs:docs_cloud/sdk_index>`.
+
+:std:doc:`D-Wave System Documentation <sysdocs_gettingstarted:index>` describes
+D-Wave quantum computers and `Leap <https://cloud.dwavesys.com/leap/>`_ hybrid solvers,
+including features, parameters, and properties. It also provides guidance
+on programming the D-Wave system, including how to formulate problems and configure parameters.
 
 Example
 =======
 This example solves a small example of a known graph problem, minimum
 `vertex cover <https://en.wikipedia.org/wiki/Vertex_cover>`_\ . It uses the NetworkX
 graphic package to create the problem, Ocean's :std:doc:`dwave_networkx <oceandocs:docs_dnx/sdk_index>`
-to formulate the graph problem as a BQM, and dwave-system's
+to formulate the graph problem as a :term:`BQM`, and dwave-system's
 :class:`~dwave.system.samplers.DWaveSampler()` to use a D-Wave system as the sampler.
-(Access to a D-Wave system has been :std:doc:`set up <oceandocs:overview/dwavesys>` in
-a configuration file that is used implicitly.) dwave-system's
-:class:`~dwave.system.composites.EmbeddingComposite()` handles mapping between the problem graph
-to the D-Wave system's numerically indexed qubits, a mapping known as :term:`minor-embedding`.
+dwave-system's :class:`~dwave.system.composites.EmbeddingComposite()` handles mapping
+between the problem graph to the D-Wave system's numerically indexed qubits,
```

Comment:
I think the sentence implies that the main benefit of the EmbeddingComposite is relabelling

Reply:
The end of this sentence is ", a mapping known as minor-embedding" which links to an explanation of minor embedding. It is a good point that I'll be taking into consideration when I use this phrasing in the future (I've been using it for a while) but here I think is not the place to get into details about embedding.